octodns_edgecenter/__init__.py

```diff
@@ -443,30 +526,63 @@ def _should_ignore(self, record):
                     want_type,
                 )
                 return True
-        if want_filters == 2:
-            return False
-        limits = [filters[i].get("limit", 1) for i in [1, 2]]
-        if limits[0] != limits[1]:
+
+        limits = {fls.get("limit") for fls in filters if "limit" in fls}
+        if limits and len(limits) != 1:
             self.log.info(
-                "ignore %s, filters.1.limit (%d) != filters.2.limit (%d)",
-                name,
-                limits[0],
-                limits[1],
+                "ignore %s, filters have different limit values", name
             )
             return True
+
         return False
 
+    @staticmethod
+    def _params_for_failover(record: Record) -> Dict:
```

Comment:
If `TCP` and/or `UDP` aren't supported, don't see mentions of them, but maybe they're just falling through `else`s then https://github.com/octodns/octodns-route53/pull/94 has an example of how to error out b/c of the lack of support.

Reply:
Our provider supports the same protocols as Octodns. So I think everything is fine. There is no need to raise exceptions.